app/models/iiif_canvas_presenter.rb

```diff
@@ -40,7 +40,11 @@ def display_content
   end
 
   def sequence_rendering
-    supplemental_files_rendering(master_file) + waveform_rendering
+    supplemental_files_rendering(master_file)
+  end
+
+  def see_also
+    waveform_see_also
```

Comment:
How about folding `waveform_see_also` into this method?  A separate method doesn't really seem necessary anymore.

Reply:
I can go either way on it. I was thinking leave it separate so we can just add additional methods if we add other stuff to the seeAlso section later, rather than have to pull the waveform back into its own method. But that's also not really any amount of extra work. 

I guess best practice would probably be to simplify the code for now, since we don't have any immediate plans for adding more to seeAlso?